Sources/SQLiteNIO/SQLiteConnection.swift

```diff
@@ -303,21 +222,10 @@ public final class SQLiteConnection: SQLiteDatabase {
     }
 }
 
-struct UnsafeTransfer<Wrapped>: @unchecked Sendable {
-    var wrappedValue: Wrapped
-    init(_ wrappedValue: Wrapped) { self.wrappedValue = wrappedValue }
-}
-extension UnsafeTransfer: Equatable where Wrapped: Equatable {}
-extension UnsafeTransfer: Hashable where Wrapped: Hashable {}
-
 final class UnsafeMutableTransferBox<Wrapped>: @unchecked Sendable {
```

Comment:
I think we could use `NIOLoopBoundBox` in these instances but open to keeping what we have

Reply:
@gwynne either way `Wrapped` should be `Sendable` otherwise mutating it is even less safe. Since `SQLLiteRow` is already `Sendable`, you could just add the `Sendable` constraint.